### ISSUE TITLE
Move servers to the collective

### ISSUE THREAD

**Issue by zimbatm** [nix-community/infra#440]
I would like to move the funding from GitHub to Open Collective to be more transparent. Open Collective also allows acquiring virtual credit cards, so we could use that to pay for them. We also have an admin@ address that we can use to forward billing emails.

**Comment by zimbatm** [comment]
The Virtual Credit Cards don't work for that use-case, so we need one member to put the money forward and then get re-imbursed though the OpenCollective expenses every month.

**Comment by zowoq** [comment]
Just to make sure I'm not misunderstanding: the servers would be on a nix-community account but the account would need to use a members card?

**Comment by Mic92** [comment]
@zimbatm I checked the situation and I think given that you live in Suiss, you won the (VAT) tax lottery, sorry :) I think you should act as the fiscal host and than re-imburse from the open-collective.

**Comment by Mic92** [comment]
> Just to make sure I'm not misunderstanding: the servers would be on a nix-community account but the account would need to use a members card?

One needs a bank account to pay the bills and than re-imburse from the opencollective.

**Comment by zowoq** [comment]
> > Just to make sure I'm not misunderstanding: the servers would be on a nix-community account but the account would need to use a members card?
> 
> One needs a bank account to pay the bills and than re-imburse from the opencollective.

I wasn't asking about bank accounts, I meant the hetzner account?

**Comment by Mic92** [comment]
> I wasn't asking about bank accounts, I meant the hetzner account?

That would probably become some nix-community account with @zimbatm payment details.

**Comment by zimbatm** [comment]
Ok, I will create a new account for admin@nix-community.org and put my Swiss debit card in it.

**Comment by zimbatm** [comment]
you can transfer your servers to K0557407123

**Comment by zowoq** [comment]
> Ok, I will create a new account for [admin@nix-community.org](mailto:admin@nix-community.org) and put my Swiss debit card in it.

Thanks for doing that, appreciated. 

Do we have any other options for getting a nix-community card that we can use for this?

**Comment by zowoq** [comment]
> Do we have any other options for getting a nix-community card that we can use for this?

Sorry for repeating the question but your previous comment doesn't address this?

**Comment by Mic92** [comment]
> > Do we have any other options for getting a nix-community card that we can use for this?
> 
> Sorry for repeating the question but your previous comment doesn't address this?

I don't think we have currently any legal body that could own a credit card except for opencollective which does not allow us to have cards.
